fontforge/fontviewbase.c

```diff
@@ -929,6 +929,108 @@ void FVAddExtrema(FontViewBase *fv, int force_adding ) {
     ff_progress_end_indicator();
 }
 
+void FVAddInflections(FontViewBase *fv, int anysel) { 
```

Comment:
I diffed these and they're identical except for the (reused) strings and the function call. The code base has that sort of thing but in this case I think its worth adding the string and function pointer to the signature of one of these and having the others be one-liners.

Reply:
Your suggested generalized solution is surely preferable. I will implement it as soon as I have a little bit more time. This affects `FVHarmonize` and `FVBalance` as well.

Reply:
@skef I am a little stuck here. I have tried your suggestion with a generalized function
`void _FVElementAction(FontViewBase *fv, int anysel, void (*func)(SplineChar*, SplineSet*, enum ae_type, int), const char* progressmsg)` that calls `func(sc, sc->layers[layer].splines, anysel, emsize)`. However my three functions `SplineCharAddInflections()`, `SplineCharBalance()` and `SplineCharHarmonize()` only take 3 parameters, whereas `SplineCharAddExtrema()` takes one additional parameter `emsize`. I think there is no elegant solution of optional parameters in C. There is of course a cheap solution: `void _FVElementAction(FontViewBase *fv, int anysel, const char* functionname)` that contains a `switch (functionname)`. Is there something I am obviously missing?